src/h5web/visualizer/Visualizer.test.tsx

```diff
@@ -1,32 +1,138 @@
 import React from 'react';
-import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import {
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+  within,
+} from '@testing-library/react';
 import { mockValues } from '../../packages/lib';
 import App from '../App';
 import { prepareForConsoleError, renderApp } from '../test-utils';
 import { MockProvider } from '../../packages/app';
+import { Vis } from '../visualizations';
+
+async function findVisSelectorTabs() {
+  return within(
+    await screen.findByRole('tablist', { name: 'Visualization' })
+  ).getAllByRole('tab');
+}
 
 describe('Visualizer', () => {
-  test('visualise scalar dataset', async () => {
+  test('show "Raw" tab for dataset with complex type', async () => {
```

Comment:
```suggestion
  test('show only "Raw" tab for dataset with complex type', async () => {
```

Reply:
Hmm if I specify it there, I'd have to specify it in the other tests as well. To me, it's not the main part of the test, a bit like the `aria-selected` check, so it doesn't have to be in the test's name.

Reply:
I would agree but the `Raw` vis is a bit 

> Hmm if I specify it there, I'd have to specify it in the other tests as well. 
Not really. The `Raw` vis is special in the way that it is shown only when there are not other vis so it is a bit of a special case.

Anyhow, it is not that important :smile_cat:

Reply:
Yeah, so you could argue that the other tests should also state that _only_ the tabs listed should be shown (i.e. that `Raw` is not shown).

Reply:
True. Let's keep it as it is, then !